common/src/main/java/earth/terrarium/adastra/api/events/AdAstraEvents.java

```diff
@@ -108,6 +132,26 @@ static boolean post(ServerLevel level, LivingEntity entity) {
         }
     }
 
+    @FunctionalInterface
+    public interface EntityGravityEvent {
+        Optional<Float> tick(Entity entity, float gravity);
+
+        static void register(EntityGravityEvent listener) {
+            ENTITY_GRAVITY_LISTENERS.add(listener);
+        }
+
+        @ApiStatus.Internal
+        static Optional<Float> post(Entity entity, float gravity) {
```

Comment:
this should be returning `float` because you pass the initial `gravity` in.

Reply:
that makes sense.
changed both.